### ISSUE TITLE
Couple of issues reported with MyBB 1.8

### ISSUE THREAD

**Issue by Cu8eR** [Sama34/MySupport#4]
http://community.mybb.com/thread-164124.html

just to take in mind

**Comment by Sama34** [comment]
I don't believe this will be added back. Why can't you use a moderation tool?

Only reason I can think of is letting users add/remove their own threads, but that could give them more power that desire (i.e: being able to mess with your index if they decide to remove all of their threads).
